geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -256,7 +257,7 @@ app.DrawController = function($scope, ngeoDecorateInteraction,
         if (evt.selected.length > 0) {
           var feature = evt.selected[0];
 
-          if (feature.get('__source__') == 'mymaps') {
+          if (!goog.string.isEmptySafe(feature.get('__map_id__'))) {
```

Comment:
This function is deprecated. Can't you safely use `if (feature.get('__map_id__'))` here?